_security-analytics/usage/rules.md

```diff
@@ -17,7 +17,27 @@ When you open the Rules page, all rules are listed in the table. Use the search
 Alternatively, you can use the **Rule type**, **Rule severity**, and **Source** dropdown menus to drill down in the list of alerts and filter for preferred results. You can use all three menus in combination to narrow results. Select only one option per menu.
```

Comment:
> Select only one option per menu.

I must have missed this during my initial read of the documentation. The filter menus for this table support selecting multiple options (e.g., the table can be filtered to display only `Critical`, and `Low` severity rules).

Reply:
Operator error. Yes, pick as many as you'd like. Thanks. Reworded this statement.